pkg/serverstatusrequest/process.go

```diff
@@ -29,23 +29,31 @@ import (
 	velerov1api "github.com/heptio/velero/pkg/apis/velero/v1"
 	"github.com/heptio/velero/pkg/buildinfo"
 	velerov1client "github.com/heptio/velero/pkg/generated/clientset/versioned/typed/velero/v1"
-	"github.com/heptio/velero/pkg/plugin/clientmgmt"
 	"github.com/heptio/velero/pkg/plugin/framework"
 )
 
 const ttl = time.Minute
 
+type PluginLister interface {
```

Comment:
We should only need the `List` func here - any particular reason why you included `DiscoverPlugins` and `Get` also?

Reply:
Because this and `Get` are part of the `PluginLister` interface. The compiler fails w/o them.

Reply:
Ohhh NVM! Fixing.